src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -181,7 +197,36 @@
 short-summary: Create a new policy for the given BackupManagementType and workloadType.
 examples:
   - name: Create a new policy for the given BackupManagementType and workloadType.
-    text: az backup policy create --policy {policy} --resource-group MyResourceGroup --vault-name MyVault --name MyPolicy
+    text: az backup policy create --policy {policy} --resource-group MyResourceGroup --vault-name MyVault --name MyPolicy --backup-management-type AzureStorage
+"""
+
+helps['backup protectable-item'] = """
+type: group
+short-summary: An item which can be protected or backed up to an Azure Recovery services vault with an associated policy.
+"""
+
+helps['backup protectable-item initialize'] = """
+type: command
+short-summary: This command triggers the discovery of any unprotected items of the given workload type in the given container. Use this command to manually discover new DBs and proceed to protect them.
```

Comment:
Refine your short summary and examples name as above

Reply:
@sambitratha : Change this to "Trigger the discovery of any unprotected items of the given workload type in the given container"